btb/tuning/metamodels/gaussian_process.py

```diff
@@ -28,7 +28,10 @@ class GaussianProcessMetaModel(BaseMetaModel):
     }
 
     def __init_metamodel__(self, length_scale=1):
-        self._MODEL_KWARGS_DEFAULT['kernel'] = RBF(length_scale=length_scale)
+        if self._model_kwargs is None:
+            self._model_kwargs = {}
+
+        self._model_kwargs['kernel'] = RBF(length_scale=length_scale)
```

Comment:
Do we need to make a deepcopy here?
I'm not sure about it, but I would double check to make sure that the same dictionary cannot be share by two different instances with different kernels.

Reply:
```python
from btb.tuning import GPTuner, IntHyperParam, Tunable

tunable = Tunable({'a': IntHyperParam()}

tuner = GPTuner(tunable, length_scale=999)
another_tuner = GPTuner(tunable, length_scale=111)
yet_another_tuner = GPTuner(tunable)

print(tuner._model_kwargs)
{'kernel': RBF(length_scale=999)}

print(another_tuner._model_kwargs)
{'kernel': RBF(length_scale=111)}

print(yet_another_tuner._model_kwargs)
{'kernel': RBF(length_scale=0.1)}
```